Travel Project v1/Interfaces/ICoordinates.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Interfaces
+{
+    public interface ICoordinates
```

Comment:
I don't get what is the purpose of this interface, could you please explain?

Reply:
It's not needed now